Gemfile

```diff
@@ -12,3 +12,4 @@ group :memcheck do
   gem "ruby_memcheck", platform: %i[mri mswin mingw x64_mingw]
 end
 gem "rbs", platform: %i[mri mswin mingw x64_mingw]
+gem "steep", platform: %i[mri mswin mingw x64_mingw]
```

Comment:
`steep` seems to have many dependencies, how about making it part of a `group :types` or so, and also move `rbs` there? So it's possible to build prism without needing to install all these gems (some are C exts).